nemo_skills/pipeline/utils.py

```diff
@@ -657,6 +682,67 @@ def get_sandox_command(cluster_config):
     return "/entrypoint.sh && /start.sh"
 
 
+def configure_client(
```

Comment:
Probably best to keep this in utils as well (we really should split that file though) and then make use of it inside generate.py, eval.py, start_server.py

Reply:
oh, but there is still configure_client in pipeline/generate. Should that one be removed then? Also can it be directly used in eval.py and check_contamination.py?

Reply:
Hmm that will requite a more refactors. I think it should be a drop in replacement, however the utils one has extra arg `get_random_port` and the generate one has extra unused arg (generation type and server port). So Id like to keep this as a separate pr item (or you can update either later on).

Reply:
But yes technically all 3 server configure calls can be merged into the utils one (if it accepts the union of all extra args from each script?)

Reply:
ok, please add a TODO or an issue for this